src/components/NavigationBar/index.js

```diff
@@ -589,11 +589,11 @@ class NavigationBar extends Component {
                             src={userAvatar}
                             size="32"
                           />
-                          {userSettingsLoaded && (
-                            <UserDetail>
-                              {!userName ? email : userName}
-                            </UserDetail>
-                          )}
+                          {/* {userSettingsLoaded && ( */}
```

Comment:
Don't do this check at the reducer level why `userSettingsLoaded` is not giving `true` value

Reply:
Are you working on the issue?